notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>if you add a <code>;</code> at the end you suppress the weird plot return value being printed in the ouptut.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>:thumbsup: (I wish there were emojis here)</p>

Reply:
Done!